### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by foolhardy21**
## Home Page.

This PR has home page APIs and UI components. Along with this I have added search videos and sort by latest feature. 

 - Files to review:
   - [ ] src/components/Home/HomeHeader.js
   - [ ] src/components/Home/CategorySection.js
   - [ ] src/components/Home/CategoryCard.js
   - [ ] src/components/Videos/VideosHeader.js
   - [ ] src/components/Videos/VideosFilter.js

- Use this link for preview - https://deploy-preview-8--pensive-hodgkin-f2cf81.netlify.app/

v-1
Thanks.

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! The code was clean and easy to understand.

File: src/components/Home/HomeHeader.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                        menu
+                    </Icon>
+
+                </Button>
+
+                {
+                    isSmallNavVisible &&
+                    <NavBar id='nav-small' classes={`flx-column pd-md ${getBgColor(theme)} pos-absolute z-index tl-0 b-solid`}>
+
+                        <Button onClick={toggleNavVisibility} classes='btn-txt mg-btm-xs'>
```
Comment (abhisheksahani-nova) [inline src/components/Home/HomeHeader.js:74]: Instead of classes , we should be using className

**Review by abhisheksahani-nova — APPROVED** [review]
Clean and readable code , nothing much to suggest ,seems good to merge

File: src/components/Home/HomeHeader.js (hunk 1 above)
Comment (foolhardy21) [reply src/components/Home/HomeHeader.js:74]: I have built these reusable components that accept `classes` argument for `className`.

**Pull request merged**